apps/payments/nextjs/package.json

```diff
@@ -36,6 +36,8 @@
     "devDependencies": {
         "@types/react": "18.2.21",
         "@types/react-dom": "18.2.7",
+        "@zerodev/passkey-validator": "5.4.0",
```

Comment:
I've made a ticket to capture this investigation. This is our workaround for now. 
WAL-2732: https://linear.app/crossmint/issue/WAL-2732/fix-appspaymentsnextjs-app-build-error